extensions/amp-ad-exit/0.1/test/test-amp-ad-exit.js

```diff
@@ -127,11 +146,31 @@ describes.realWin('amp-ad-exit', {
     win.document.body.appendChild(adDiv);
   }
 
+  /**
+   * Add a response
+   * @param {!Window} win
+   * @param {!string} vendor The identifier for the third-party frame that
+   * responded
+   * @param {!Object<string,string>} response The response object sent from
+   * the third-party vendor's iframe
+   */
+  function addToResponseMap(win, vendor, response) {
+    win[AMP_ANALYTICS_3P_RESPONSES] =
+        win[AMP_ANALYTICS_3P_RESPONSES] || {};
+    win[AMP_ANALYTICS_3P_RESPONSES][vendor] = response;
```

Comment:
We are overwriting the last response from same vendor here. Is that what we what, or this is only for testing purpose?

Reply:
We want to save the most recent response from a certain vendor to a certain creative. So, I've not yet addressed your previous comment a few lines above, but assuming all is well there, then this line here is WAI.